build.sbt

```diff
@@ -244,15 +244,18 @@ lazy val restoreGCS = project
     librarySettings,
     name := s"$baseName-restore-gcs"
   )
-  .dependsOn(coreRestore, coreGCS)
+  .dependsOn(coreRestore % "compile->compile;test->test", coreGCS % "compile->compile;test->test")
```

Comment:
This change is necessary because otherwise the restore CLI fails with

```
[WARN ] ManifestInfo - You are using version 10.2.7 of Akka HTTP, but it appears you (perhaps indirectly) also depend on older versions of related artifacts. You can solve this by adding an explicit dependency on version 10.2.7 of the [akka-http-spray-json] artifacts to your project. Here's a complete collection of detected artifacts: (10.1.11, [akka-http-spray-json]), (10.2.7, [akka-http, akka-http-core, akka-http-testkit, akka-http-xml, akka-parsing]). See also: https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed
```

Note that this is the exact same the backup equivalent at https://github.com/aiven/guardian-for-apache-kafka/blob/main/build.sbt#L166-L173